hack/setup-build-index-image.sh

```diff
@@ -72,6 +72,7 @@ spec:
 
           git clone --single-branch --branch OPERATOR_RELEASES https://github.com/openshift/special-resource-operator.git
           cd special-resource-operator/bundle/OPERATOR_VERSION/
+	  rm manifests/image-references
```

Comment:
Is this a file or directory? If a directory, it will require -r parameter

Reply:
it's a file this is just a backport :)